app/models/game.rb

```diff
@@ -100,4 +100,34 @@ def piece_at(target_x_axis, target_y_axis)#destination of piece
    self.pieces.where(:x_axis => target_x_axis, :y_axis => target_y_axis).first
   end
 
+ def find_path(position_1, position_2) # check if proposed move is obstructed
```

Comment:
Why not use `is_obstructed?`.  We should probably refactor so it's **DRY** code.